lib/rspec-puppet/support.rb

```diff
@@ -228,9 +228,15 @@ def build_catalog(*args)
     end
 
     def munge_facts(facts)
-      output = {}
-      facts.keys.each { |key| output[key.to_s] = facts[key] }
-      output
+      return facts.reduce({}) do | memo, (k, v)|
```

Comment:
Sorry I missed this before my week off.

When I was talking about the string literal I meant not using the `family = 'RedHat'` variable. Unless you want to iterate over multiple values, using a variable instead of replicating the value is just obscuring what's going on.